internal/compiler/passes/inlining.rs

```diff
@@ -165,6 +165,17 @@ fn inline_element(
 
     core::mem::drop(elem_mut);
 
+    root_component.setup_code.borrow_mut().extend(
```

Comment:
I think the visit_all_named_references_in_element should normally take care of that, because you should adjust visit_element_expressions to also visit the init code, this is important.  This would solve the other pass i think

Reply:
Yes, you're right - I can remove the mapping part. Still need to copy the init code though.